lms/static/sass/base/_base.scss

```diff
@@ -130,7 +130,6 @@ a:focus {
   margin: 0 auto 0;
   padding: ($baseline*2) 0;
   max-width: grid-width(12);
-  min-width: 760px;
```

Comment:
Thanks, @marjev. Yeah, I would override this property on your view-specific Sass file (rather than in the _base.scss file). I'd recommend creating new Sass partials in /lms/static/sass/views/ for the homepage as well as the course results views. Anything specific to those can be contained in there and scoped to selectors those views have (i.e. ``.container-course``).